### PR TITLE
Sr add v2 billing endpoints

### PR DESCRIPTION
**Pull request by s-rubenstein**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -241,6 +241,240 @@ paths:
...
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ErrorReport'
+        500:
+          description: Internal Server Error
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ErrorReport'
```
Comment (gpolumbo-broad) [inline src/main/resources/swagger/api-docs.yaml:343]: Was there an intentional decision to decide to leave out `/api/billing/v2/{projectId}/billing-account`?  What was the reasoning behind deciding which APIs were added to Orch and which were not?

**Review by marctalbott — APPROVED** [review]

File: src/main/resources/swagger/api-docs.yaml (hunk 1 above)
Comment (s-rubenstein) [reply src/main/resources/swagger/api-docs.yaml:343]: There wasn't an intentional decision, I just did all the ones I could find in the api-docs in Rawls. Let me double check.

Comment (s-rubenstein) [reply src/main/resources/swagger/api-docs.yaml:343]: Ah! Its cause that endpoint doesn't exist in Rawls yet, only on our feature branch.

Comment (marctalbott) [reply src/main/resources/swagger/api-docs.yaml:343]: Updated the ticket to possibly cherry-pick that endpoint onto develop to include a note to update Orch as well

https://broadworkbench.atlassian.net/browse/CA-1140

**Review by gpolumbo-broad — APPROVED** [review]

**Pull request merged**